nf_core/commands_test_datasets.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import click
+import rich
+
+from nf_core.test_datasets.list import list_dataset_branches, list_datasets
+from nf_core.test_datasets.search import search_datasets
+from nf_core.utils import rich_force_colors
+
+log = logging.getLogger(__name__)
+stdout = rich.console.Console(force_terminal=rich_force_colors())
+
+# Files / directories starting with one of the following in a git tree are ignored:
+IGNORED_FILE_PREFIXES = [
+    ".",
+    "CITATION",
+    "LICENSE",
+    "README",
+    "docs",
+]
+
+
+def test_datasets_list_branches(ctx: click.Context) -> None:
+    """
+    List all branches on the nf-core/test-datasets repository.
+    Only lists test data and module test data based on the curated list
+    of pipeline names [on the website](https://raw.githubusercontent.com/nf-core/website/refs/heads/main/public/pipeline_names.json).
+    """
+    list_dataset_branches()
+
+
+def test_datasets_list_remote(
+    ctx: click.Context, maybe_branch: str, generate_nf_path: bool, generate_dl_url: bool
```

Comment:
Just a suggestion, but I like `branch` more 😄 to match the name of the parameter
```suggestion
    ctx: click.Context, branch: str, generate_nf_path: bool, generate_dl_url: bool
```

Reply:
Good point - Done. I kept the naming in the downstream methods, because it reflects the need to maybe prompt for the argument first.